lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -500,10 +500,15 @@ def max_thread_limit
   end
 
   sha = `git rev-parse HEAD`[0..7]
+  branch = `git branch`
 
   # If we're in a CI environment include our build number
   if build = ENV['BUILD_NUMBER'] || ENV['TRAVIS_BUILD_NUMBER']
-    new_version = sprintf('%s-%04d-%s', version, build, sha)
+    if branch.include? "* release"
```

Comment:
Should we really be special casing release branch? Can we make this generic?

e.g. for master, append nothing. for any other branch append, say the first 7 letters or something?